selfie.c

```diff
@@ -468,6 +468,13 @@ void setValue(uint64_t* entry, uint64_t value)        { *(entry + 5) = value; }
 void setAddress(uint64_t* entry, uint64_t address)    { *(entry + 6) = address; }
 void setScope(uint64_t* entry, uint64_t scope)        { *(entry + 7) = scope; }
 
+// --------------------------- HASH TABLE --------------------------
+
+uint64_t GLOBAL_HASH_TABLE_SIZE = 1024;
```

Comment:
I'm not completely sure, if exchanging a dynamic data structure with a static structure is good in this case, especially if you don't handle errors. Possible solutions would be
-  Grow your hash table size, as you need it (maybe with a log(n) complexity in the number of entries)
-  Detect when the symbol table is full or reaches a specific load factor (because performance suffers from an high load factor) and exit with an error message